docs/iris/src/whatsnew/contributions_2.4.1/bugfix_2020-Feb-14_pp_emptyslices.txt

```diff
@@ -0,0 +1,5 @@
+* Fixed a problem which was causing file loads to fetch *all* field data
```

Comment:
Why have you put this as a a 2.4.1 bug fix? Surely its going into Iris 2.4.0?

Reply:
Ooops.
I thought we already *had* a whatsnew/2.4.rst, but now I see we don't.
So I now think this was just a mistake + will attempt a fix ..